cmd/defenseclaw-setup/authenticode.go

```
@@ -461,6 +524,19 @@ func countAttributeValues(values asn1.RawValue) (int, error) {
 
 func inspectEmbeddedAuthenticode(filePath string) (embeddedAuthenticodeMetadata, error) {
 	pkcs7, present, err := readEmbeddedPKCS7(filePath)
+	return inspectEmbeddedAuthenticodePayload(pkcs7, present, err)
+}
+
+func inspectEmbeddedAuthenticodeFromFile(file *os.File) (embeddedAuthenticodeMetadata, error) {
```

golangci-lint (2.12.2)

[error] func inspectEmbeddedAuthenticodeFromFile is unused

(unused)